nucypher/characters/lawful.py

```diff
@@ -511,7 +509,7 @@ def _pick_treasure_map(self, treasure_map=None, map_id=None):
             else:
                 raise ValueError("You need to pass either treasure_map or map_id.")
         elif map_id:
-                raise ValueError("Don't pass both treasure_map and map_id - pick one or the other.")
+            raise ValueError("Don't pass both treasure_map and map_id - pick one or the other.")
```

Comment:
We need to form a convention for this language - Perhaps even an exception class.

Reply:
You mean for this XOR-like argument passing?